core/slice.go

```diff
@@ -289,8 +321,13 @@ func (sl *Slice) generateSlicePendingHeader(block *types.Block, newTermini []com
 		return types.PendingHeader{}, err
 	}
 
+	copyTermini := make([]common.Hash, 4)
```

Comment:
What are the four elements of this array?

Reply:
deleted. thanks, its an awesome non-overloaded perfectly understandable self-explanatory array for any competent programmer.

Reply:
I agree...